content/docs/gke/gcp-e2e.md

```diff
@@ -532,19 +532,20 @@ Next, upload the container image to Container Registry so that you can run it on
     kustomize edit add configmap mnist-map-training   --from-literal=learningRate=0.01
     ```
 
-1. Configure parameters and save the model to Cloud Storage:
+1. Configure parameters for where the training results and exported model will be saved in Cloud Storage.  We'll use a timestamp-based subdirectory so that if the tutorial is run more than once, the training will start fresh each time.
```

Comment:
Style hint: Avoid the use of "we" in tech docs, as it's not clear who the "we" refers to. Use "you" instead. Also avoid the use of future tense, as some non-English languages don't have future tense in the same way as English, and readers get confused and wonder when exactly this thing will happen. (Full details in the [style guide](https://www.kubeflow.org/docs/about/style-guide/#address-the-audience-directly).

Reply:
done, though if it doesn't read well to you feel free to make additional suggestions :)